internal/js/modules/k6/browser/common/network_manager.go

```diff
@@ -558,15 +558,16 @@ func (m *NetworkManager) onRequest(event *network.EventRequestWillBeSent,
 		m.logger.Errorf("NetworkManager", "creating request: %s", err)
 		return
 	}
-	// Skip data and blob URLs, since they're internal to the browser.
```

Comment:
Of course this would break `TestURLSkipRequest`, as that test is precisely preventing regressions on these lines.

Reply:
Arguably this would be a breaking change, so we might need to consider carefully, and perhaps leave it up to k6 v2.0, if we really want to ship it.